packages/ui-components/src/Containers/NavBar/index.tsx

```diff
@@ -54,6 +54,15 @@ const NavBar: FC<PropsWithChildren<NavbarProps>> = ({
           className={style.sidebarItemTogglerLabel}
           htmlFor="sidebarItemToggler"
           role="button"
+          tabIndex={0}
+          onKeyDown={e => {
+            if (e.key === 'Enter' || e.key === ' ') {
```

Comment:
I'm -1 of this change. I'm unsure of what you're trying to achieve here, but this menu is only visible on mobile screens, which very often are touch-only. Screen readers should already be accessible to this with `aria-label`, there's no need for an onKeyDown here.